### ISSUE TITLE
[Supporters] Convert each image to a link to that organization.  (Explicitly requested by several orgs).

### ISSUE THREAD

**Issue by kbonnen** [NeuromatchAcademy/NeuromatchAcademy.github.io#15]

**Comment by maryiii** [comment]
Do we have any descriptions for them? What should be included in separate pages? Links should go to another page or a pop-up will be enough?

**Comment by kbonnen** [comment]
@maryiii For each image we need:
- alt text: company or organization name
- a link to that organization's website (open that link in a new page)

Does that answer your question?

**Comment by maryiii** [comment]
@kbonnen Yes got it! I misunderstood the task, I thought we need a separate page for each organization on our website. So we just need to link organizations to their websites. Just a question. I checked the organizations logos and almost all of them contain the name, So is it necessary to add text or linking logo the the site would be enough?

**Comment by kbonnen** [comment]
I don't think we need the names with the images but it would be good to add alt-text (hover text) to the images if google sites allows that.

**Comment by maryiii** [comment]
@kbonnen Kate we have two options here:
1- Simply adding the organization's url to our image and write their name as alt text(for increasing the accessibility of our website). If you check the sponsors page, the logo under presenting sponsor is like this.
2- Our second option is through loading the image with an HTML embedded piece of code. In this case, if you hover your mouse on the logo, organization's name would appear under the logo. In this case we will have the organization's name as an alt text as well. The only difference is that we need to have a piece of code for all logos and the images should be uploaded on our drive. (The FRL logo on the left of the presenting sponsor is our second option)
Please let me know which one is more suitable for our page, and I will finalize the task.

**Comment by maryiii** [comment]
@kbonnen 
Any comments on this?

**Comment by kbonnen** [comment]
I think I prefer the alt text alone.  (option 1)

Is that thought with option 2 that it makes it feel more like something you could click on?

**Comment by maryiii** [comment]
Yes exactly. So I will make all the logos clickable then we can decide if it is needed to add some effects on it. In terms of implementation, option 1 is easier as well.

**Comment by maryiii** [comment]
@kbonnen 
Link + alt text is added for all of the organizations except think:theory, I couldn't find its website.
